docs/source/development/commands_reference.md

```diff
@@ -390,6 +402,10 @@ The following runs all `pytest` unit tests found in `src/tests`, including cover
 kedro test
 ```
 
+___
```

Comment:
For the `kedro test` and `kedro lint` commands we can link to the new doc pages.

Reply:
I agree this is a good idea, though I would prefer to wait until #1904 is merged, if that sounds OK.